libraries/connectivity/metrics/phaselockingvalue.h

```diff
@@ -118,8 +118,10 @@ class CONNECTIVITYSHARED_EXPORT PhaseLockingValue : public AbstractMetric
     *
     * @return                   The connectivity information in form of a network structure.
     */
-    static Network phaseLockingValue(const QList<Eigen::MatrixXd> &matDataList, const Eigen::MatrixX3f& matVert,
-                                     int iNfft=-1, const QString &sWindowType="hanning");
+    static Network phaseLockingValue(const QList<Eigen::MatrixXd> &matDataList,
+                                     const Eigen::MatrixX3f& matVert,
+                                     int iNfft=-1,
+                                     const QString &sWindowType="hanning");
```

Comment:
const QString &sWindowType = QStringLiteral("hanning") would reduce the number of internal type conversions.

Reply:
will change this at a later point